packages/editor/src/components/parts/program-start/JavaClassSelector.tsx

```diff
@@ -0,0 +1,30 @@
+import { Combobox, ComboboxItem, useFieldset } from '../../../components/widgets';
+import { useEditorContext, useMeta } from '../../../context';
+import { Consumer } from '../../../types/lambda';
+import { PathFieldset } from '../common';
+import { Type } from '@axonivy/inscription-protocol';
+
+type JavaClassSelectorProps = {
+  config: string;
+  onChange: Consumer<string>;
+  type: Type;
+};
+
+const JavaClassSelector = ({ config, onChange, type }: JavaClassSelectorProps) => {
+  const { context } = useEditorContext();
+  const javaClassItems = [
+    ...useMeta('meta/program/types', { type: type, context }, []).data.map<ComboboxItem>(javaClass => {
+      return { value: javaClass.fullQualifiedName };
+    })
+  ];

```

Comment:
you already get here an array back from the meta request and you don't have to destructure it and "repack" it into an array again. Simply do:
`const javaClassItems = useMeta('meta/program/types', { type: type, context }, []).data.map<ComboboxItem>(javaClass => ({ value: javaClass.fullQualifiedName }));`

The destructure / "repack" is a cool syntax if you directly want to add multiple stuff into an array.